code/game/objects/items/stacks/rods.dm

```diff
@@ -6,6 +6,9 @@ GLOBAL_LIST_INIT(rod_recipes, list ( \
 	new/datum/stack_recipe("railing", /obj/structure/railing, 3, time = 18, window_checks = TRUE), \
 	new/datum/stack_recipe("tank holder", /obj/structure/tank_holder, 2, time = 5, one_per_turf = TRUE, on_floor = FALSE), \
 	new/datum/stack_recipe("ladder", /obj/structure/ladder/crafted, 15, time = 150, one_per_turf = TRUE, on_floor = FALSE), \
+	//SKYRAT EDIT ADDITION BEGIN - WRESTLING RING
+	new/datum/stack_recipe("wrestling ropes", /obj/structure/railing/wrestling, 3, time = 18, window_checks = TRUE), \
+	//SKYRAT EDIT ADDITION END
```

Comment:
Don't put comments inside extended line long lists, put it after or before.
It fucks up the compiler hard

Reply:
i mean, there are countless other comments just like that in the list for recipes